sdk/core/azure-core/azure/core/pipeline/transport/__init__.py

```diff
@@ -24,137 +24,92 @@
-                pass  # Trio not installed
-
-            try:
-                from ._aiohttp import AioHttpTransport, AioHttpTransportResponse
-
-                __all__.extend([
-                    'AioHttpTransport',
-                    'AioHttpTransportResponse',
-                ])
-            except ImportError:
-                pass  # Aiohttp not installed
-    except (ImportError, SyntaxError):
-        # requests library is installed but asynchronous pipelines not supported.
-        pass
-except (ImportError, SyntaxError):
-    # requests library is not installed
-    try:
-        from ._base_async import AsyncHttpTransport, AsyncHttpResponse
-        __all__.extend([
-            'AsyncHttpTransport',
-            'AsyncHttpResponse',
-        ])
-
-        if sys.version_info >= (3, 7):
-            __all__.extend([
-                'AioHttpTransport',
-                'AioHttpTransportResponse',
-            ])
-
-            def __dir__():
-                return __all__
 
-            def __getattr__(name):
-                if name == 'AioHttpTransport':
-                    try:
-                        from ._aiohttp import AioHttpTransport
-                        return AioHttpTransport
-                    except ImportError:
-                        raise ImportError("aiohttp package is not installed")
-                if name == 'AioHttpTransportResponse':
-                    try:
-                        from ._aiohttp import AioHttpTransportResponse
-                        return AioHttpTransportResponse
-                    except ImportError:
-                        raise ImportError("aiohttp package is not installed")
-                return name
+        
+def __dir__():
+    return __all__
 
-        else:
-            try:
-                from ._aiohttp import AioHttpTransport, AioHttpTransportResponse
-                __all__.extend([
-                    'AioHttpTransport',
-                    'AioHttpTransportResponse',
-                ])
-            except ImportError:
-                pass  # Aiohttp not installed
-    except (ImportError, SyntaxError):
-        pass  # Asynchronous pipelines not supported.
+def __getattr__(name):
+    if name == 'AsyncioRequestsTransport':
+        try:
+            from ._requests_asyncio import AsyncioRequestsTransport
+            return AsyncioRequestsTransport
+        except ImportError:
+            pass
+    if name == 'AsyncioRequestsTransportResponse':
+        try:
+            from ._requests_asyncio import AsyncioRequestsTransportResponse
+            return AsyncioRequestsTransportResponse
+        except ImportError:
+            pass
+    if name == 'AsyncHttpTransport':
+        try:
+            from ._base_async import AsyncHttpTransport
+            return AsyncHttpTransport
+        except ImportError:
+            pass
+    if name == 'AsyncHttpResponse':
+        try:
+            from ._base_async import AsyncHttpResponse
```

Comment:
This has no reason to fail as this is a protocol, should be line 28 and not in a block like that